src/layout/Section.jsx

```diff
@@ -47,6 +51,10 @@ Section.propTypes = {
 	noSeparator: PropTypes.oneOfType([
 		PropTypes.bool,
 		PropTypes.oneOf(Object.keys(VALID_BREAKPOINTS))
+	]),
+	flushUntil: PropTypes.oneOfType([
+		PropTypes.bool,
```

Comment:
feels weird to support a boolean because it's not immediately clear what this means:

`<Section flushUntil>...`

`<Section className="flush--horizontal">` may be more useful for the use case of flushing at all breakpoints. This doesn't exist in sasstools yet, but it's an easy addition...